src/main/java/org/wildfly/security/password/util/ModularCrypt.java

```diff
@@ -70,13 +59,7 @@ private PasswordUtil() {}
     private static final int A_APACHE_HTDIGEST          = 7;
     private static final int A_BSD_CRYPT_DES            = 8;
     private static final int A_CRYPT_DES                = 9;
-    private static final int A_DIGEST_MD2               = 10;
```

Comment:
We could actually make it completely generic.  If the password type isn't recognized as a modular crypt type, then we pass it to the password spec encoder, which then uses ``convertibleToKeySpec`` to determine if the password can be encoded.  Then new passwords automatically get modular crypt support too (albeit not under a standard name).